Documentation/Contributing/ci-configuration.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CI Configuration
+---
+
+## Environment Variables / Secrets
```

Comment:
This list of secrets looks a bit different from what we have in the rook repo. And aren't they just secrets defined in the repo, not env vars?